centreon/www/front_src/src/Dashboards/Dashboard/Layout/Panel/MorePanelActions.tsx

```diff
@@ -58,6 +59,7 @@ const MorePanelActions = ({
   const { openModal } = useWidgetForm();
 
   const edit = (): void => {
+    isEditing(true);
```

Comment:
```suggestion
    canEdit && isEditing(true);
```
Then call the function when the user has edit role